src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgery.cs

```diff
@@ -325,10 +325,21 @@ private bool IsValidationRequired(HttpContext httpContext)
             return true;
         }
 
+        private static IAntiforgeryContextAccessor GetAntiforgeryFeature(HttpContext httpContext)
```

Comment:
Was `IAntiforgeryFeature` intentional and you want to create an interface with all the properties here or did you just mean `AntiforgeryFeature`.